lambdas/issue-credential/tests/issue-credential-handler.test.ts

```diff
@@ -71,6 +72,59 @@ const testAnswerResultHappy = {
       },
     ],
     verificationScore: 2,
+    ci: ["CI1"],
+    checkDetailsCount: 3,
+    failedCheckDetailsCount: 0,
+  },
+};
+
+const testAnswerResultHappyFailedCheckDetails = {
+  Item: {
+    correlationId: "correlationId",
+    ttl: 1234567890,
+    answers: [
+      {
+        questionKey: "rti-p60-payment-for-year",
+        status: "incorrect",
+      },
+      {
+        questionKey: "rti-payslip-income-tax",
+        status: "incorrect",
+      },
+      {
+        questionKey: "sa-income-from-pensions",
+        status: "incorrect",
+      },
+    ],
+    verificationScore: 2,
```

Comment:
If all questions are incorrect should verification score not be 0

Reply:
Yeah your right good spot